experimental/dds/tree2/src/core/rebase/verifyChangeRebaser.ts

```diff
@@ -4,7 +4,7 @@
  */
 
 import { makeAnonChange } from "./changeRebaser";
-import { ChangeRebaser } from ".";
+import { ChangeRebaser, revisionMetadataSourceFromInfo } from ".";
```

Comment:
That will fix the build error
```suggestion
import { makeAnonChange } from "./changeRebaser";
import { ChangeRebaser, revisionMetadataSourceFromInfo } from ".";
```